sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClient.cs

```diff
@@ -144,7 +144,7 @@ public AzureOpenAIClient(Uri endpoint, TokenCredential credential, AzureOpenAICl
     /// <param name="endpoint"> The endpoint to use. </param>
     /// <param name="options"> The additional client options to use. </param>
     protected AzureOpenAIClient(ClientPipeline pipeline, Uri endpoint, AzureOpenAIClientOptions options)
-        : base(pipeline, endpoint, options)
+        : base(pipeline, endpoint, null)
```

Comment:
I don't feel strongly about this, but want to make sure we've considered the full implications of not using the same client options as the inherited client.  Do we maintain separate pipelines and/or how is pipeline-construction handled?  Could there be a problematic scenario in the future if the base client needs to do something internal with its options type and the derived client doesn't have visibility into that?

Reply:
(Very delayed comment response, but adding here for posterity) -- we've discussed this at length (including with Krzysztof) and believe it's right to split them. Pipeline instantiation is handled distinctly via the protected internal constructor path; AOAI uses AOAI client options to make an AOAI-configured pipeline for clients to use.

The big thing is that this allows OpenAI concepts with no current applicability to Azure OpenAI (like ProjectId) to be properly represented in the OpenAI options _without_ having an extraneous and confusing reappearance in the Azure layer.